classes/PrestaShopLogger.php

```diff
@@ -163,7 +169,7 @@ public static function eraseAllLogs()
      *
      * @return true if exists
      */
-    protected function _isPresent()
+    protected function isPresent()
```

Comment:
You cannot modify the name of protected or public method, you have to at least alias it. It breaks travis build.